tests/Security/Handler/AclSecurityHandlerTest.php

```diff
@@ -143,21 +143,6 @@ public function testAddObjectOwnerParamMustBeMutableAclInterface(): void
         $handler->addObjectOwner($this->createStub(AclInterface::class));
     }
 
-    public function testUpdateAclMustOnlyAcceptMutableAclInterface(): void
```

Comment:
This test fails on github actions, not sure if it makes much sense to test this TBH. On my local env it triggers a warning, but on github actions it triggers en exception. I have check the php ini config and it looks the same on both envs (assert config)